deviceModeInit.js

```diff
@@ -538,36 +677,6 @@ var rudderTracking = (function () {
       });
   }
 
-  /**
-   * Checks whether the url for the product contains the variant id
-   * @returns {string} variant id or null
-   */
-  function findVariantIdInURL() {
-    const matches = window.location.href.match(/\d{8,20}/);
-    if (matches) {
-      return matches[0]
-    }
-    return null;
-  };
-
-  /**
-   * Returns the sku value for the variant matching the id in url 
-   * @param {*} payload product payload generated using product mapping
-   * @returns {string} matching variant's sku or undefined
-   */
-  function getVariantSku(payload) {
```

Comment:
Not needed anymore as this was used to get sku from variants objects array. But now it is only an object and can simply be retreived